web/src/app/rotations/util.js

```diff
@@ -112,3 +112,12 @@ export function reorderList(_items, oldIndex, newIndex) {
   items.splice(newIndex, 0, _items[oldIndex]) // add dest to newIndex position
   return items
 }
+
+export function getHours(count, unit) {
+  const lookup = {
+    hourly: 1,
+    daily: 24,
+    weekly: 24 * 7,
+  }
+  return lookup[unit] * count
+}
```

Comment:
If it lives here it should have documentation on what it does, supported units, etc... but feels like noise -- it's only used in one place and spreads the code/logic out unnecessarily.